### PR TITLE
Bugfix if multiple interfaces are present

### PR DESCRIPTION
**Pull request by trick77**
- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

_1_
_Fix if multiple network interfaces are available_

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

**Comment by DL6ER** [comment]
Would this add multiple interfaces to the config file? Like
```
PIHOLE_INTERFACE=eth0 eth1 eth2
```
?

**Comment by trick77** [comment]
Yes, that's exactly what it does unless in quotes.

**Comment by DL6ER** [comment]
@trick77 Yeah, sorry, I guess that was more of an rhetorical question. Something like
```
PIHOLE_INTERFACE=eth0 eth1 eth2
```
is bad and will lead to a failure of `dnsmasq`

**Comment by trick77** [comment]
Not sure what you're saying but the current code just doesn't work.

**Comment by DL6ER** [comment]
Sorry, I misinterpreted what you were changing where.

Approved

**Comment by trick77** [comment]
Thanks!!

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -235,7 +235,7 @@ chooseInterface() {
   # Loop sentinel variable
   local firstLoop=1
 
-  if [[ $(echo ${availableInterfaces} | wc -l) -eq 1 ]]; then
+  if [[ $(echo "${availableInterfaces}" | wc -l) -eq 1 ]]; then
       PIHOLE_INTERFACE=${availableInterfaces}
```
Comment (DL6ER) [inline automated install/basic-install.sh]: Add `" "` here as well. Not necessary here, but consistent style.

**Comment by trick77** [comment]
Done.

**Comment by Mcat12** [comment]
This has already been fixed in master as a hotfix: #1116

**Comment by Mcat12** [comment]
Approved. We can merge this so that master and dev are back in sync, because of the hotfix.

**Pull request merged**